tutorials/llm/llama-3/pruning-distillation/04_a_distilling_depth_pruned_student.ipynb

```diff
@@ -0,0 +1,136 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "09d30e35-8e9d-4d2e-bd14-738c627a3963",
+   "metadata": {},
+   "source": [
+    "### Step 4: Distill knowledge from teacher into student\n",
+    "Distillation of a model with NeMo Framework is also possible using a python script: [megatron_gpt_distillation.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_distillation.py). In this notebook, we will explore distillation with the depth-pruned model as the `STUDENT` model. \n",
+    "\n",
+    "For this demonstration, the `TEACHER` would be the finetuned teacher model `megatron_llama_ft.nemo` and the `STUDENT` model would be the pruned 4B model. This training run is capped by `STEPS`, and validation is carried out every `VAL_INTERVAL` steps."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c33cf641-0d27-417f-b3ee-c06701698184",
+   "metadata": {},
+   "source": [
+    "#### Step 4.a.: Using depth-pruned student\n",
+    "While distilling knowledge from the teacher to depth-pruned model, the `STUDENT` model would be  `4b_depth_pruned_model.nemo` as produced by the [depth-pruning](./03_a_depth_pruning.ipynb) notebook. This training run is capped by `STEPS`, and validation is carried out every `VAL_INTERVAL` steps.\n",
+    "\n",
+    "> `NOTE:` In the block of code below, pass the paths to your pre-processed train, test and validation data files as well as path to the teacher and student .nemo models."
```

Comment:
fix punctuation

`NOTE:` In the block of code below, pass the paths to your pre-processed train, test, and validation data files, as well as path to the teacher and student .nemo models."